test/blocks/vimeo/vimeo.test.html

```diff
@@ -0,0 +1,39 @@
+<html>
+  <head>
+    <!-- the import map to use in our test -->
+    <script type="importmap">
+      {
+        "imports": {
+          "../../../libs/utils/utils.js": "./mocks/utils.js"
+        }
+      }
+    </script>
+  </head>
+
+  <body>
+    <a href="https://vimeo.com/636596252">https://vimeo.com/636596252</a>
+  </body>
+
+  <script type="module">
+    import { runTests } from '@web/test-runner-mocha';
+
```

Comment:
you have this newline after `import { runTests } from '@web/test-runner-mocha` in each test file. Any reason as to why?